### PR TITLE
chore: test on node10

### PR DESCRIPTION
**Pull request by alexander-fenster**
This PR brings an updated configuration file for CircleCI:

- test on Node.js v10
- more YAML references to make config more readable (at least I hope so)

This is an automated PR prepared using [github-repo-automation](https://github.com/googleapis/github-repo-automation).

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -219,7 +197,7 @@ jobs:
     working_directory: /home/node/dlp-samples
   system_tests:
     docker:
-      - image: 'node:8'
+      - image: 'node:10'
```
Comment (JustinBeckwith) [inline .circleci/config.yml]: I wouldn't change this just yet.  8 is still latest LTS.

File: .circleci/config.yml [hunk 2]
```diff
@@ -250,7 +221,7 @@ jobs:
           when: always
   publish_npm:
     docker:
-      - image: 'node:8'
+      - image: 'node:10'
```
Comment (JustinBeckwith) [inline .circleci/config.yml]: Same as above, I'd leave this on 8 until 10 goes LTS.

File: .circleci/config.yml (hunk 1 above)
Comment (alexander-fenster) [reply .circleci/config.yml]: sounds good - will revert those back to node8

**Comment by alexander-fenster** [comment]
Thanks @JustinBeckwith - I fixed that, any other feedback please? :)

**Comment by alexander-fenster** [comment]
@callmehiphop Not right now - we can only drop Node4 support within the major release, and we actually don't want to do it right now and will better wait a little bit more (@ofrobots can comment further)

**Pull request merged**